<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget_of_Isuue</class>
 <widget class="QWidget" name="Widget_of_Isuue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>71</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>371</width>
    <height>71</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>完成</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="DescriptionLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>280</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>280</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>280</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="TiTleLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="HourNumber">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>20</height>
    </size>
   </property>
   <property name="smallDecimalPoint">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="PositionLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="ColonLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>10</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>10</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>10</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLabel" name="RepeatLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>  TextLabel</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="MinNumber">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>20</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
